app/services/bill-runs/two-part-tariff/fetch-match-details.service.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+/**
+ * Fetches the individual charge element match details during the review stage of a two-part tariff bill run
+ * @module FetchMatchDetailsService
+ */
+
+const BillRunModel = require('../../../models/bill-run.model.js')
+const ReviewChargeElementModel = require('../../../models/review-charge-element.model.js')
+
+/**
+ * Fetches the match details for an individual charge element
+ *
+ * @param {String} billRunId - UUID of the bill run
+ * @param {String} licenceId - UUID of the individual licence to review
+ * @param {String} reviewChargeElementId - The UUID of the review charge element being viewed
+ *
+ * @returns {Promise<Object[]>} Contains an array of bill run data and review charge element data
```

Comment:
Correction. We're not returning an array and neither value is an array.

```suggestion
 * @returns {Promise<Object>} An object containing the bill run and review charge element instances
```